src/main/java/ru/hogwarts/school/controller/StudentController.java

```diff
@@ -1,35 +1,42 @@
 package ru.hogwarts.school.controller;
 
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 import ru.hogwarts.school.model.Student;
-import ru.hogwarts.school.service.StudentService;
+import ru.hogwarts.school.service.StudentServiceImpl;
 
 import java.util.List;
 
 @RestController
-@RequestMapping("student")
+@RequestMapping("/student")
 public class StudentController {
 
-    private final StudentService service;
+    private final StudentServiceImpl service;
```

Comment:
```suggestion
    private final StudentService service;
```
инъекцию надо делать по интерфейсам, а не реализациям, чтобы хоть где-то был смысл в использовании интерфейсов для сервисов